Containerfile.istio-csr

```diff
@@ -0,0 +1,43 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_golang_1.22 AS builder
```

Comment:
@bharath-b-rh Shall we name the file `Containerfile.cert-manager-istio-csr` to be consistent with the repo name?

Reply:
Yeah +1

Reply:
Should we change name of image as well to https://github.com/openshift/cert-manager-operator-release/pull/94/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R62 `cert-manager-istio-csr` ?

Reply:
also submodule name to same convention?

Reply:
yeah, sounds good.